### PR TITLE
Pretty impl FromStr for WebSocketAccept and WebSocketKey

### PR DESCRIPTION
**Pull request by kpp**

### CODE REVIEW

File: src/header/accept.rs [hunk 1]
```diff
@@ -26,20 +26,13 @@ impl FromStr for WebSocketAccept {
...
-						"Sec-WebSocket-Accept must be 20 bytes",
-					));
-				}
-				let mut array = [0u8; 20];
-				let mut iter = vec.into_iter();
-				for i in &mut array {
-					*i = iter.next().unwrap();
-				}
-				Ok(WebSocketAccept(array))
+				use std::convert::TryInto;
```
Comment (vi) [inline src/header/accept.rs]: Thi s bumps Minimial-Supported-Rust-Version from approximately 1.30 all the way up to 1.34. Even Debian Testing's version of Rust is currently 1.32.

Is it that much needed?

Comment (kpp) [reply src/header/accept.rs]: No. I will try another solution

Comment (kpp) [reply src/header/accept.rs]: fixed

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,12 @@
 language: rust
-rust: nightly
+rust: nightly-2019-04-19
```
Comment (vi) [inline .travis.yml]: Why specific version?

Comment (kpp) [reply .travis.yml]: Because sometimes there are no clippy/fmt shipped with a new nightly.

**Comment by kpp** [comment]
@vi ping

**Comment by vi** [comment]
@kpp, pong; not doing much Rust now;

Maybe I should just switch `websocket`'s travis to stable instaead of nightly.

**Comment by kpp** [comment]
We will. But we have to fix tests.

**Pull request merged**